setup.py

```diff
@@ -16,7 +16,7 @@
 
 setup(
     name='ccm',
-    version='2.0.5',
+    version='2.0.6',
```

Comment:
why changing this ? i.e. we are not uploading CCM anywhere, no point of changing this

Reply:
is this causing any problems? 
I mean, when I reinstall ccm I always see 2.0.5 and i wonder if there was some change...

Reply:
It's irrelevant to this change

Since we don't have releases.

We can open an issue for starting doing so, I would love putting CCM up in pypi, it can help us in lots of places.

But for now we stick to branches.

Reply:
fixed.
I also consider puting ccm to pypi as a good idea.